src/app/lib/utilities/moment/index.stories.jsx

```diff
@@ -89,100 +89,664 @@ const editorialWhitelist = [
 ];
 
 const methods = [
-  { what: '[default format]', method: locale => moment(A_DATE).locale(locale).format() },
-  { what: 'MMM Do YYYY', method: locale => moment(A_DATE).locale(locale).format("MMM Do YYYY") },
-  { what: 'MMMM Do YYYY, h:mm:ss a', method: locale => moment(A_DATE).locale(locale).format('MMMM Do YYYY, h:mm:ss a') },
-  { what: 'YYYY [escaped text] YYYY', method: locale => moment(A_DATE).locale(locale).format('YYYY [escaped text] YYYY') },
-  { what: 'LT', method: locale => moment(A_DATE).locale(locale).format('LT') },
-  { what: 'LTS', method: locale => moment(A_DATE).locale(locale).format('LTS') },
-  { what: 'l', method: locale => moment(A_DATE).locale(locale).format('l') },
-  { what: 'L', method: locale => moment(A_DATE).locale(locale).format('L') },
-  { what: 'll', method: locale => moment(A_DATE).locale(locale).format('ll') },
-  { what: 'LL', method: locale => moment(A_DATE).locale(locale).format('LL') },
-  { what: 'lll', method: locale => moment(A_DATE).locale(locale).format('lll') },
-  { what: 'LLL', method: locale => moment(A_DATE).locale(locale).format('LLL') },
-  { what: 'llll', method: locale => moment(A_DATE).locale(locale).format('llll') },
-  { what: 'LLLL', method: locale => moment(A_DATE).locale(locale).format('LLLL') },
+  {
```

Comment:
This file has `/* eslint-disable prettier/prettier */` on line 70 to ensure that linters dont correct this file, making it far far too long. Perhaps your editor did this?

Can you reset this please, for this case, fewer longer lines is much much easier to read than 700 shorter ones :)